src/cascadia/TerminalApp/App.cpp

```diff
@@ -1307,8 +1315,15 @@ namespace winrt::TerminalApp::implementation
                                             _settings->GlobalSettings().GetDefaultProfile();
         const auto controlSettings = _settings->MakeSettings(realGuid);
 
-        // Create a Conhost connection based on the values in our settings object.
-        TerminalConnection::ITerminalConnection controlConnection = TerminalConnection::ConhostConnection(controlSettings.Commandline(), controlSettings.StartingDirectory(), 30, 80, winrt::guid());
+        TerminalConnection::ITerminalConnection controlConnection{ nullptr };
+        if (controlSettings.Commandline() == L"Azure")

```

Comment:
The magic string L"Azure" is used in multiple places. It should be defined once as a constant imported into all useful locations.

(This will still be applicable if you change it to an enum or guid)

Reply:
> Also, we will want to find a solution that doesn't require us to commit the entire VCPKG nuget package into the repository since it's 50+MB.

I recently discovered I had to use `vcpkg` to install `cpprestsdk:x64-windows`.

One solution would be to move away from submodules to git subtrees.
https://blog.developer.atlassian.com/the-power-of-git-subtree/

This would reduce the burden of a submodule which by default pulls in the entire repo (and its history).

Reply:
I love subtrees, but.. we're not using submodules for cpprest or anything vcpkg-related.